book/02-git-basics/sections/remotes.asc

```diff
@@ -198,12 +198,14 @@ $ git remote show origin
 ----
 
 Dieser Befehl zeigt an, zu welchem Zweig automatisch gepusht wird, wenn Sie `git push` ausführen, während Sie sich in bestimmten Branches befinden.
-Er zeigt Ihnen auch, welche entfernten Branches auf dem Server sind, die Sie noch nicht haben, welche entfernten Branches Sie haben, die aber vom Server entfernt wurden und die lokalen Branches, die automatisch mit ihrem Remote-Tracking-Branch mergen können, wenn Sie `git pull` ausführen.
+Er zeigt Ihnen auch, welche entfernten Branches auf dem Server sind, die Sie noch nicht haben, welche entfernten Branches Sie haben, die aber vom Server entfernt wurden und die lokalen Branches, die automatisch mit Ihrem Remote-Tracking-Branch zusammengeführt (gemergt) werden können, wenn Sie `git pull` ausführen.
 
-==== Umbenennen und entfernen von Remotes
+==== Umbenennen und Entfernen von Remotes
 
-Sie können `git remote rename` ausführen, um den Kurznamen einer Fernbedienung zu ändern.(((Git Befehle, remote)))
-Wenn Sie beispielsweise `pb` in `paul` umbenennen möchten, können Sie das mit `git remote rename` machen:
+Sie können `git remote rename` ausführen, um den Kurznamen eines Remotes zu ändern.(((Git Befehle, remote)))
+// Wiederholung auch im englischen Original. Besser:
+// "... können Sie dies mit diesem Befehl machen:"
```

Comment:
Was ist mit diesem Kommentar?

Reply:
Sollte gelöscht werden

Reply:
Soll also so mit der Wiederholung bleiben? Warten wir noch ab, ob @cbachhuber dies im englischen Repo melden möchte.

Reply:
???
Eben stehe ich auf dem Schlauch – das Löschen bezog sich auf die Kommentarzeilem.
In der Folgezeile hast du meinen Vorschlag angenommen, oder?

Reply:
Oh, ja klar, sorry.

```suggestion
```